website/data/docs-nav-data.json

```diff
@@ -651,6 +651,10 @@
         "title": "Configuration",
         "path": "nia/configuration"
       },
+      {
+        "title": "License",
```

Comment:
What do you think about naming it `"Enterprise License"` in the navbar? The Consul navbar has "Consul Enterprise" -> and then "License" as a subnav.

For CTS, there's not a dedicated section for CTS enterprise, and renaming "License" may help with the enterprise association.

Reply:
Another option could be to build out a new "Enterprise" subnav then -> License. I can see how a dedicated "Ok this is a list of enterprise features available" in the future.

Reply:
Agree, I was also thinking that an enterprise page would be helpful! I'll add an Enterprise section with an overview page that links to the new driver and then move the license page be a child page

Reply:
I added an Enterprise subnav and within the subnav moved the licensing page, linked the Terraform Cloud driver page, and added an Overview page. Let me know if you have feedback! Thank you